dev-resources/puppetlabs/trapperkeeper/services/authorization/auth.conf

```diff
@@ -1,15 +0,0 @@
-authorization: {
- rules: [
- {
-   path: /path/to/resource
-   type: path
-   allow: [ "*.domain.org", "*.test.com" ]
-   deny: "bad.guy.com"
- },
- {
-   type: regex
-   path: "(incoming|outgoing)"
-   allow: "www.domain.org"
- }
- ]
-}
```

Comment:
These files were never used.  We can add test fixtures in when we write the tests that use them.